frontend/app/src/store/assets/nft.ts

```diff
@@ -0,0 +1,79 @@
+import { type ActionResult } from '@rotki/common/lib/data';
+import { getDomain } from 'tldts';
```

Comment:
Is this the only function we use from the `tldts` library? any reason why we need a new dependency and why we can't use a regex for the same purpose?

Reply:
regex doesn't work because some url can have subdomain, and have more than 1 part suffix (.co.id)
the worst case looks like this 
Input: `https://subdomain.domain.co.id`
Output: `domain.co.id`

If we want to make it right, we need to list all possible suffixes `.co.id`, `.co.uk`.

The best way we can do with regex is only get this output:
`subdomain.domain.co.id`

The problem is if we have multiple image with same domain but different subdomain, we need to allow it separately.

Reply:
How does the library do it? is it overly complicated?

Maybe we could do it ourselves? It feels a bit weird importing a huge library just for a method.